packages/react/src/ResourceDiffRow/ResourceDiffRow.tsx

```diff
@@ -0,0 +1,73 @@
+import { useState } from 'react';
+import { Table, Button } from '@mantine/core';
+import { ResourcePropertyDisplay } from '../ResourcePropertyDisplay/ResourcePropertyDisplay';
+import { InternalSchemaElement, TypedValue } from '@medplum/core';
+import classes from './ResourceDiffRow.module.css';
+
+export interface ResourceDiffRowProps {
+  name: string;
+  path: string;
+  property: InternalSchemaElement | undefined;
+  originalValue: TypedValue | undefined;
+  revisedValue: TypedValue | undefined;
+  shouldToggleDisplay: boolean;
+}
+
+export function ResourceDiffRow(props: ResourceDiffRowProps): JSX.Element {
+  const { name, path, property, originalValue, revisedValue, shouldToggleDisplay } = props;
+  const [isCollapsed, setIsCollapsed] = useState(true);
```

Comment:
The default value here should be `true` if the `property` is an `Attachment` type, and `false` otherwise.

We can check if the `property` is an `Attachment` like this:

```ts
property?.type?.find((t) => t.code === 'Attachment')
```

So:
```suggestion
  const isAttachmentType = property?.type?.find((t) => t.code === 'Attachment');
  const [isCollapsed, setIsCollapsed] = useState(isAttachmentType);
```